tests/experimental/test_utils.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+
+from bigchaindb.common.crypto import generate_key_pair
+from bigchaindb.models import Transaction
+
+
+pytestmark = pytest.mark.tendermint
+
+
+def generate_create_and_transfer(keypair=None):
+    if not keypair:
+        keypair = generate_key_pair()
+    priv_key, pub_key = keypair
+    create_tx = Transaction.create([pub_key], [([pub_key], 10)]).sign([priv_key])
+    transfer_tx = Transaction.transfer(
+            create_tx.to_inputs(),
+            [([pub_key], 10)],
+            asset_id=create_tx.id).sign([priv_key])
+    return create_tx, transfer_tx
+
+
+def test_validation_worker_process_multiple_transactions(b):
```

Comment:
If I understand this test correctly, you set up one worker and have it process all the txs. I'd also like to see a test with 2 workers, where `worker_1` processes `transfer_tx` and `worker_2` processes `double_spend`, and we verify that the parallel processes still catch the double spend.

Reply:
I see your point, but given the Routing Algorithm™ (that is nothing more than the modulo operator) all transactions of a specific `asset_id` are routed to the same worker.

I'll try to write a test around it.